docs/en/core/writing-modes.md

```diff
@@ -0,0 +1,16 @@
+# Writing Modes
+In the View Menu, Zettlr features several Editor settings that may enhance your writing experience. Have a play and see what works for you!
+
+## Night Mode
+Also commonly known as 'Dark Mode', clicking this option will change the background colour of the Editor to black and the default text colour to white.
+
+## Distraction-Free Mode
+The distraction free mode removes visual clutter from the Editor environment. Zettlr opens with the [File Manager sidebar](file-manager.md) on the left side of the screen, showing your Workspaces and Files. If you now click on 'Distraction free' in the View menu, you will notice the sidebar disappear and your text content shift towards the centre of the screen.
+
+In this mode, the sentence you're currently working on is highlighted and completed sentences are dimmed. Clicking anywhere on a sentence, shifts the cursor position and highlights that sentence.
```

Comment:
The dimming is an option that can be disabled

Reply:
Fixed

Reply:
Also reworded explanation of Typewriter mode.